deploy/create-export-url-bucket.sh

```diff
@@ -0,0 +1,33 @@
+  #!/bin/bash -xb
+#
+# Creates GCS bucket for Export to Saturn feature.
+#
+# See export_url_controller.py for more information.
+#
+# jq, gsutil must be installed before running this script.
```

Comment:
optional: could add a check for this and print out an error saying install `jq` or `gsutil`

Reply:
If jq and gsutil aren't installed, output is:
```
melchang@melchang:~/data-explorer$ deploy/create-export-url-bucket.sh 1000_genomes
deploy/create-export-url-bucket.sh: line 18: jq: command not found
deploy/create-export-url-bucket.sh: line 21: gsutil: command not found
Creating gs://-export
deploy/create-export-url-bucket.sh: line 27: gsutil: command not found
Setting bucket TTL to 1 day
deploy/create-export-url-bucket.sh: line 30: gsutil: command not found
Setting up CORS
deploy/create-export-url-bucket.sh: line 33: gsutil: command not found
```
I don't think it's worth detecting if they're installed, because the output is already pretty clear that they should be installed.